core/tests/protractor_utils/editor.js

```diff
@@ -397,6 +397,30 @@ var addParameterChange = function(paramName, paramValue) {
   general.waitForSystem(500);
 };
 
+// This function adds a exploration level parameter change, creating
+// the parameter if necessary.
+var addExplorationLevelParameterChange = function(paramName, paramValue) {
+  runFromSettingsTab(function() {
+    element(by.css('.protractor-test-exploration-edit-param-changes')).click();
+    element(by.css('.protractor-test-add-param-button')).click();
+
+    var editorRowElem = element.all(by.css(
+    '.protractor-test-param-changes-list')).last();
+
+    forms.AutocompleteDropdownEditor(editorRowElem).setValue(paramName);
+
+    /* Setting parameter value is difficult via css since the associated
+    input is a sub-component of the third party select2 library. We isolate
```

Comment:
@giritheja Sean is right. The parameter name is controlled by `select2` but parameter value is a plain input field. So, please make sure to update your comment. The actual problem is the input fields are dynamically generated and they don't have classes defined, so there's no clean way of accessing the elements. 

Why not access the input field using `editorRowElem.all(by.tagName('input')).last()`? 
(I personally don't like the use of magic number 2 and one has to dig into the code to see why it is the third input field.)

What do you think?